<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_Page</class>
 <widget class="QDialog" name="Search_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>715</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1070</width>
    <height>715</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1070</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(236, 240, 241);
}
QPushButton::hover{
color:white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2001</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color:rgb(255,0,54);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border-image: url(:/Final_Logo.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>2001</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(50,50,50);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/home.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>16</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/search.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cloud">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
	border-image: url(:/cloud-computing.png);
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="notification">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>16</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/notification.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="user">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>14</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/man-user.png);
border-radius:10px;
min-height:20px;
min-width:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="enter_username_lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:10px;
padding-left:10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>                  Enter the userid </string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:20px;
background-color:rgb(52, 152, 219);
font: 14pt &quot;Ubuntu&quot;;
}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="make_friend_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:20px;
background-color:rgb(52, 152, 219);
font: 14pt &quot;Ubuntu&quot;;
}</string>
   </property>
   <property name="text">
    <string>Friend</string>
   </property>
  </widget>
  <widget class="QLabel" name="display_username_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="display_profile_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_profile_button">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>340</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:20px;
background-color:rgb(52, 152, 219);
font: 14pt &quot;Ubuntu&quot;;
}</string>
   </property>
   <property name="text">
    <string>View Profile</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>frame</zorder>
  <zorder>label_2</zorder>
  <zorder>enter_username_lineEdit</zorder>
  <zorder>search_button</zorder>
  <zorder>make_friend_button</zorder>
  <zorder>display_username_label</zorder>
  <zorder>display_profile_label</zorder>
  <zorder>view_profile_button</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
